pkg/cmd/attestation/verification/tuf.go

```diff
@@ -23,6 +25,9 @@ func DefaultOptionsWithCacheSetting() *tuf.Options {
 		opts.DisableLocalCache = true
 	}
 
+	// Set the cache path to a directory owned by the CLI
+	opts.CachePath = filepath.Join(config.CacheDir(), ".sigstore", "root")
```

Comment:
Let's make sure we bump `go-gh` in a separate PR to get https://github.com/cli/go-gh/pull/153 pulled in.

Reply:
https://github.com/cli/cli/pull/8884